daprdocs/content/en/reference/components-reference/supported-pubsub/setup-hazelcast.md

```diff
@@ -7,6 +7,8 @@ aliases:
   - "/operations/components/setup-pubsub/supported-pubsub/setup-hazelcast/"
 ---
 
+> **Deprecation notice**: The Hazelcast PubSub component has been deprecated due to inherent lack of support for "at least once" delivery guarantee, and will be removed in a future Dapr release.
```

Comment:
Have this notice within an alert block ? See the warning alert block below.

Reply:
Updated as requested